great_expectations/datasource/fluent/spark_azure_blob_storage_datasource.py

```diff
@@ -152,7 +152,7 @@ def _build_data_connector(
             container=abs_container,
             name_starts_with=abs_name_starts_with,
             delimiter=abs_delimiter,
-            file_path_template_map_fn=AzureUrl.AZURE_BLOB_STORAGE_HTTPS_URL_TEMPLATE.format,
+            file_path_template_map_fn=AzureUrl.AZURE_BLOB_STORAGE_WASBS_URL_TEMPLATE.format,
```

Comment:
This was correction of a typographical error (causing incorrect conversion of the path for Spark utilization -- it was using the one for Pandas -- incorrectly -- instead).